src/news/2017-08-30-self-assessment-toolkit.md

```diff
@@ -23,8 +23,7 @@ The following images show examples of the designs we are currently developing
 (select to enlarge):
 
 <figure>
-<a href="images/Self-reflection-Figure-1.png"><img src="images/Self-reflection-Figure-1
--thumb.png" alt="view larger version of this image" aria-details="det1"></a>
+<a href="images/Self-reflection-Figure-1.png"><img src="images/Self-reflection-Figure-1-thumb.png" alt="view larger version of this image" aria-details="det1"></a>
```

Comment:
This line is failing linting, due to line length being greater than 120 characters.

Reply:
This is an issue with linting Markdown the includes inline HTML; the URL breaks if the line is split but the Markdown linter fails otherwise. What's your preference in terms of how to approach this, @jobara?

Reply:
@greatislander in this case, you might be able to just uncollapse the markup.

```suggestion
<a href="images/Self-reflection-Figure-1.png">
    <img src="images/Self-reflection-Figure-1-thumb.png" alt="view larger version of this image" aria-details="det1">
</a>
```